hapic/ext/pyramid/context.py

```diff
@@ -106,20 +106,24 @@ def get_file_response(self, file_response: HapicFile, http_code: int):
                 # is automatically guessed
                 content_type=file_response.mimetype or None,
             )
+            response.conditional_response = file_response.accept_byte_range
         else:
             from pyramid.response import FileIter
             from pyramid.response import Response
 
-            response = Response(status=http_code)
+            response = Response(status=http_code, conditional_response=file_response.accept_byte_range)
```

Comment:
not sure about the condition we should use to active conditional response, maybe we should have 2 field:
- one for accept range header only
- one for conditional response mecanism activation 
?

Reply:
i change a bit the behavior in order to support that:
- conditional response is active by default
- other framework don't support this feature now and should not try to return that their does accept bytes.